app/bundles/EmailBundle/Entity/EmailReplies.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\EmailBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Mautic\ApiBundle\Serializer\Driver\ApiMetadataDriver;
+use Mautic\CoreBundle\Doctrine\Mapping\ClassMetadataBuilder;
+use Mautic\CoreBundle\Entity\IpAddress;
+use Mautic\LeadBundle\Entity\Lead;
+
+/**
+ * Class EmailReplies.
+ */
+class EmailReplies
+{
+    /**
+     * @var int
+     */
+    private $id;
+
+    /**
+     * @var Email
+     */
+    private $email;
+
+    /**
+     * @var \DateTime
+     */
+    private $dateReplied;
+}
```

Comment:
If you add the loadMetadata() method and define the table metadata, this could be used as the cross table @escopecz suggested.